linode/manager
# feat: [M3-6733] - VPC landing page

## Description 📝
- Add VPC to side menu navigation and create menu dropdown
- Add VPC landing page and empty state

## Preview 📷
![image](https://github.com/linode/manager/assets/115299789/12e7e1f4-5fe8-43ab-9a88-1161637767ca)

![image](https://github.com/linode/manager/assets/115299789/a59d3c91-3348-4bf3-8c9a-e3305db3f15d)

## How to test 🧪
```
yarn test VPCLanding
```
Turn on the mocks and check the VPC landing page. To check the empty state, you can set line 63 of VPCLanding.tsx to true
